README.md

```diff
@@ -68,9 +68,10 @@ $ sudo yum remove eosio
 
 #### Build Script Uninstall
 
-If you have previously installed EOSIO using build scripts, you may execute `./scripts/eosio_uninstall.sh` to uninstall.
-- Passing `--force` will answer yes to all prompts
-- Passing `--full` will remove data directories (be very careful with this)
+If you have previously installed EOSIO using build scripts, you can execute `eosio_uninstall.sh` to uninstall.
+- Passing `-y` will answer yes to all prompts (forces `-f`)
```

Comment:
I don't think `-y` should force `-f`. They should be independent.

`-y` alone should not remove the data directories and not prompt the user for anything. `-f` alone should remove the data directories after prompting the user if they are sure. `-f -y` should proceed forward with removing the data directories without prompting the user.

Reply:
Ah, I misunderstood. I'll get that fixed